src/Client/Markup.fs

```diff
@@ -15,17 +15,15 @@ type Clickability =
 
 let goToUrl (e: React.MouseEvent) =
     e.preventDefault()
-    let href = !!e.target?href
+    let href = !!e.currentTarget?href
```

Comment:
target will point to an html element that was clicked, not the <a> element. So when I have nested elements in <a> and click on a <span>: target will contain <span>?href = undefined; currentTarget will contain <a>?href = correctUrl

Reply:
Alright :)